Formula/genometools.rb

```diff
@@ -0,0 +1,33 @@
+class Genometools < Formula
+  desc "GenomeTools: The versatile open source genome analysis software"
+  homepage "http://genometools.org/"
+  url "http://genometools.org/pub/genometools-1.5.10.tar.gz"
+  sha256 "0208591333b74594bc219fb67f5a29b81bb2ab872f540c408ac1743716274e6a"
+  head "https://github.com/genometools/genometools.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "cairo"
+  depends_on "pango"
+  depends_on :python
+
+  def install
+    args = ["prefix=#{prefix}"]
+    system "make", *args
+    system "make", "install", *args
+
+    cd "gtpython" do
+      # Use the shared library from this specific version of genometools.
+      inreplace "gt/dlload.py",
```

Comment:
I don't understand why an inreplace should be needed here. Is there no `make` target for python that takes into account the prefix we already specified above? What is the default behavior without the inreplace? This seems like a bug if it can't be specified via a setup option.

Reply:
The default behaviour of macOS is to search `$HOME/lib:/usr/local/lib:/usr/lib` for the shared library. The above code works when the library is installed in one of these standard locations.
https://docs.python.org/3/library/ctypes.html#loading-shared-libraries
https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man3/dlopen.3.html

Perhaps it should search the specified installation directory in addition to these default paths.

Reply:
>Perhaps it should search the specified installation directory in addition to these default paths.

Yeah.